specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkSecurityPerimeter.json

```diff
@@ -701,7 +701,169 @@
           "NspAssociations"
         ],
         "operationId": "NspAssociations_List",
-        "description": "Lists the NSP resource associations in the specified perimeter.",
+        "description": "Lists the NSP resource associations.",
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
```

Comment:
>ResourceGroupNameParameter [](http://example.com/codeflow?start=34&length=26)

Recommend using ResourceGroupName parameter in v3 common types. Same for SubscriptionId and ApiVersion parameter.

Reply:
Thanks @mentat9. All the networking resources using the same. Will update in the next PR. Will have make a lot of changes.